resources/definitions/zmorph_i500.def.json

```
@@ -0,0 +1,168 @@
+        "raft_base_line_spacing": { "value": "raft_base_line_width * 2" },
+        "raft_interface_line_spacing": { "value": "raft_interface_line_width + 0.3" },
+        "raft_interface_line_width": { "value": "0.5" },
+        "raft_interface_speed": { "value": "45" },
+        "raft_interface_thickness": { "value": "0.3" },
+        "raft_margin": { "value": "4" },
+        "raft_speed": { "value": "60" },
+        "raft_surface_layers": { "value": "2" },
+        "raft_surface_thickness": { "value": "0.2" },
+        "raft_surface_line_width": { "value": "machine_nozzle_size * 1.2" },
+        "raft_surface_line_spacing": { "value": "raft_surface_line_width * 1.1" },
+        "retraction_amount": { "value": "0.7" },
+        "retraction_count_max": { "value": "10" },
+        "retraction_extrusion_window": { "value": "1" },
+        "retraction_hop": { "value": "0.3" },
+        "retraction_hop_enabled": { "value": "extruders_enabled_count > 1"  },
+        "retraction_hop_only_when_collides": { "value": "True" },
+        "retraction_min_travel": { "value": "1.5" },
+        "retraction_prime_speed": { "value": "50" },
+        "skin_overlap": { "value": "10" },
+        "speed_equalize_flow_enabled": { "value": "True" },
+        "speed_layer_0": { "value": "20" },
+        "speed_prime_tower": { "value": "speed_topbottom" },
+        "speed_print": { "value": "40" },
+        "speed_support": { "value": "speed_wall_0" },
+        "speed_support_interface": { "value": "speed_topbottom" },
+        "speed_topbottom": { "value": "math.ceil(speed_print * 20 / 35)" },
+        "speed_wall": { "value": "math.ceil(speed_print * 30 / 35)" },
+        "speed_wall_0": { "value": "math.ceil(speed_wall * 20 / 30)" },
+        "speed_wall_x": { "value": "speed_wall" },
+        "support_angle": { "value": "51" },
+        "support_pattern": { "value": "'triangles'" },
+        "support_use_towers": { "value": "False" },
+        "support_xy_distance": { "value": "wall_line_width_0 * 2" },
+        "support_xy_distance_overhang": { "value": "wall_line_width_0" },
+        "support_z_distance": { "value": "0" },
+        "switch_extruder_retraction_speed": { "value": "70" },
+        "switch_extruder_prime_speed": { "value": "35" },
+        "switch_extruder_retraction_amount": { "value": "0" },
+        "top_bottom_thickness": { "value": "1" },
+        "travel_avoid_supports": { "value": "True" },
+        "travel_avoid_distance": { "value": "3 if extruders_enabled_count > 1 else machine_nozzle_tip_outer_diameter / 2 * 1.5" },
+        "wall_0_inset": { "value": "0" },
+        "wall_line_width_x": { "value": "line_width" },
+        "wall_thickness": { "value": "0.8" },
+        "meshfix_maximum_resolution": { "value": "(speed_wall_0 + speed_wall_x) / 60" },
+        "meshfix_maximum_deviation": { "value": "layer_height / 4" },
+        "optimize_wall_printing_order": { "value": "True" },
+        "retraction_combing": { "default_value": "all" },
```

This setting override is not working. It is overridden by the `value` defined in fdmprinter, which is a formula. This is a way to make it override properly:

```suggestion
        "retraction_combing": { "value": "'all'" },
```

Or you could consider adjusting the formula from fdmprinter to make it dependant on whether ironing or monotonic is used.